data/marameref.json

```
@@ -0,0 +1,35 @@
+{
+  "name": "Amarachi Omereife",
+  "bio": "Seasoned Software Engineer and Azure Cloud Engineer **Proud Techie**. Health & Wellness Enthusiast"
```

```suggestion
  "bio": "Seasoned Software Engineer and Azure Cloud Engineer **Proud Techie**. Health & Wellness Enthusiast",
```
There is a comma missing here.